friskby/GW/air_in_bergen.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+import requests
+import timestring
+#  import re
+#  import time
+from datetime import date
+import psycopg2
+from friskby.GW.friskby_gw import FriskByGW
```

Comment:
well I can simplify it at least to push code on github; on my laptop when I try to use just _from friskby_gw import FriskByGW_ I get an error: "Unresolved reference".